tools/c7n_gcp/c7n_gcp/resources/bigquery.py

```diff
@@ -33,6 +33,9 @@ class resource_type(TypeInfo):
         @staticmethod
         def get(client, event):
             # dataset creation doesn't include data set name in resource name.
+            if 'protoPayload' not in event.keys():
+                return client.execute_query('get', verb_arguments=event)
```

Comment:
what is this for?

Reply:
This update added for child resource, as child resource use parent get method for getting data. In this case in "event " we do not have all data like "protoPayload". Here we have only information like project id and dataset id.

Reply:
nit: key in dictionary is spelled `if 'protoPayload' in event:`

Reply:
Updated